README.md

```diff
@@ -24,11 +24,12 @@ Here is an incomplete list of our users, whose integrations may be anywhere from
 
 ### Bindings & Packages
 
-If C++ isn't your cup of tea, Manifold also has bindings to many other languages, some maintained in this repository, and others elsewhere.
+Manifold has bindings to many other languages, some maintained in this repository, and others elsewhere. It can also be built in C++ via [vcpkg](https://github.com/microsoft/vcpkg.git).
 
 | Language | Packager | Name | Maintenance |
 | --- | --- | --- | --- |
 | C | N/A | N/A | internal |
+| C++ | vcpkg | [manifold](https://github.com/microsoft/vcpkg/tree/master/ports/manifold) | internal |
```

Comment:
This says vcpkg is under internal maintenance - are we maintaining this package or are you? If it's really going to be internal, we could use a hand setting up a Github Action to publish it.

Reply:
I don't think it can be internal, we need to get vcpkg repo to approve the PR.

Reply:
Anyway, nice catch :)

Reply:
Updated to external. Misunderstood it to mean bindings written by others. Thanks!